### PR TITLE
Adds `batchInvoiceLineItems` collection.

### PR DESCRIPTION
**Pull request by dmccraw**

### CODE REVIEW

File: docs/collections/batch_invoice_line_items.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+```javascript
+// ~~~~~
+// Saves batchInvoiceLineItem item.
+teamsnap.saveBatchInvoiceLineItem(batchInvoiceLineItem);
+
+// ~~~~~
+// Creates a new batchInvoiceLineItem then saves it.
+var batchInvoiceLineItem = teamsnap.createBatchInvoiceLineItem({
+  batchInvoiceId: 1,
+  quantity: '1',
```
Comment (matthopson) [inline docs/collections/batch_invoice_line_items.md]: String?

Comment (dmccraw) [inline docs/collections/batch_invoice_line_items.md]: Good catch. Fixed in both batch_invoice_line_items.md and invoice_line_items.md

File: src/collections/batchInvoiceLineItems.coffee [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    throw new TSArgsError 'teamsnap.saveBatchInvoiceLineItem',
+      "`batchInvoiceLineItem.type` must be 'batchInvoiceLineItem'"
+  unless batchInvoiceLineItem.batchInvoiceId
+    return @reject 'You must choose a batchInvoiceId.', 'batchInvoiceId',
+      callback
+  unless batchInvoiceLineItem.quantity
+    return @reject 'You must provide a quantity.', 'quantity',
+      callback
+  unless batchInvoiceLineItem.amount
+    return @reject 'You must provide a amount.', 'amount',
```
Comment (matthopson) [inline src/collections/batchInvoiceLineItems.coffee]: "an amount"

Comment (dmccraw) [inline src/collections/batchInvoiceLineItems.coffee]: Fixed in both batch_invoice_line_items and invoice_line_items and rebuilt teamsnap.js and teamsnap.min.js

**Pull request merged**